src/edu/ucsd/library/xdre/harvesting/FieldMappings.java

```diff
@@ -212,6 +220,33 @@ private void initFieldMappings() throws Exception
         return fieldMappings;
     }
 
+    /*
+     * Retrieve cell value from merged cells if exists
+     * @param cell
+     * @param rowNum
+     * @param colNum
+     * @return
+     */
+    private String getCellValue(Cell cell, int rowNum, int colNum) {
+
+        // handle merged cell
+        for (int i = 0; i < sheet.getNumMergedRegions(); i++) {
+            CellRangeAddress region = sheet.getMergedRegion(i);
+
+            int frIndex = region.getFirstRow();
+            int fcIndex = region.getFirstColumn();
+
+            if ( rowNum >= frIndex && rowNum <= region.getLastRow() && colNum >= fcIndex && colNum <= region.getLastColumn()) {
+                // retrieve merged cell value
+                cell = sheet.getRow(frIndex).getCell(fcIndex);
```

Comment:
Is there a test you can point me to where this is being tested? I'm trying to make sure I understand the context of this new way of retrieving cell values.

Reply:
This is just for handling merged cells. I'll see how to add a test for it then.

Reply:
@mcritchlow I've added a test for retrieving values from merged cells.